### PR TITLE
add a readme

### PR DESCRIPTION
**Pull request by ryandillinfelton**
Just adds a little readme to document how to use the new cache buster script

### CODE REVIEW

File: src/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+## Akamai Cache Buster
+
+This script is run automatically from Jenkins each time a frontend is deployed
+to `Prod`. It clears out all of the old cached versions of the application to make
+sure users are served up to date content
+
+### To Run
+
+```bash
+python bustCache.py /path/to/your/.edgerc nameOfApp
```
Comment (tahmidefaz) [inline src/README.md]: It probably doesn't matter at all, but does `appName` in place of `nameOfApp` sounds better?

File: src/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+
+# Frontend Builder Common
+
+The repo responsible for building all of the frontends on cloud.redhat.com
+
+## Akamai Cache Buster
+
+This script is run automatically from Jenkins each time a frontend is deployed
+to `Prod`. It clears out all of the old cached versions of the application to make
+sure users are served up to date content
```
Comment (kruai) [inline src/README.md]: I'd change "up to date" to "up-to-date", and end the sentence with a period.

File: src/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+
+# Frontend Builder Common
+
+The repo responsible for building all of the frontends on cloud.redhat.com
```
Comment (kruai) [inline src/README.md]: Might wanna end the sentence with a period.

File: src/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+```bash
+python bustCache.py /path/to/your/.edgerc nameOfApp
+```
+
+### Some Notes and Requirements
+
+* your edgerc needs read/write permission for the eccu API on akamai (not open CCU)
+* the script only works on production akamai (no way to clear the cache on staging)
+* requests take about 30 minutes to finish
+* will not work on apps that don't have paths listed in the [source of truth](https://github.com/RedHatInsights/cloud-services-config/blob/ci-beta/main.yml)
```
Comment (kruai) [inline src/README.md]: I think this list might look better if the first word in each sentence was capitalized.

File: src/README.md (hunk 1 above)
Comment (ryandillinfelton) [reply src/README.md]: Yes

**Comment by ryandillinfelton** [comment]
> Do we want this README to be inside of `src` or the root?

Whoppsie daisy. Moved it to the root

File: README.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+
+# Frontend Builder Common
+
+The repo responsible for building all of the frontends on cloud.redhat.com.
+
+## Akamai Cache Buster
+
+This script is run automatically from Jenkins each time a frontend is deployed
+to `Prod`. It clears out all of the old cached versions of the application to make
+sure users are served up-to-date content
```
Comment (kruai) [inline README.md]: Still needs a period, but then it's good to go.

**Pull request merged**